Makefile

```diff
@@ -71,11 +71,11 @@ run: fmt check
 install: manifests
 	kubectl apply -f config/crd/bases
 
-# Deploy controller in the configured Kubernetes cluster in ~/.kube/config
+# Deploy controller to the Kubernetes cluster specified in the environment variable KUBECONFIG
+# Modify the Helm template located at charts/druid/templates if any changes are required
 .PHONY: deploy
-deploy: manifests $(KUSTOMIZE)
-	kubectl apply -f config/crd/bases
-	kustomize build config/default | kubectl apply -f -
```

Comment:
Please provide a new `Makefile` target `make deploy-via-kustomize`? Since the repo still supports kustomize-based deployments, and there might be developers who might actually prefer this method, we should not completely remove it, and atleast provide a Makefile target for this.

Reply:
Thanks, Shreyas, for nice suggestion.
Fixed as part of https://github.com/gardener/etcd-druid/pull/599/commits/231818f804186b50cbd4eb061cb139482fcf09fb.